.gitignore

```diff
@@ -1 +1,2 @@
-.idea
\ No newline at end of file
+.idea
+AwesomeAustn/desktop.ini
```

Comment:
it should ignore the file if you `**/desktop.ini`

Reply:
Oh. I used GitHub Desktop and it did it for me. Is it not right?

Reply:
it works fine but mine will just be more general for people aside from you lol